### ISSUE TITLE
AAC 3.3.1 Kicks when riding Boats

### ISSUE THREAD

**Issue by Cryptite** [konsolas/AAC-Issues#896]
On PaperSpigot 1.12.1, many players getting kicked riding boats:

### Core information
**Server version**: PaperSpigot 1.12.1

**AAC version**: 3.3.1

**ProtocolLib version**: 4.3.0

```
[25-08-2017 12:12:23.122] RedSkilZZ failed Fly: (BOAT) moved unexpectedly: 0.523 (ABOVE WATER) [ping:78 tps:20.0] VL:9
[25-08-2017 12:12:23.122] RedSkilZZ failed Fly: (BOAT) moved unexpectedly: 0.523 (ABOVE WATER) [ping:78 tps:20.0] VL:18
[25-08-2017 12:12:23.223] RedSkilZZ failed Fly: (BOAT) moved unexpectedly: 0.000 (ABOVE WATER) [ping:78 tps:19.93] VL:24
[25-08-2017 12:12:23.223] RedSkilZZ failed Fly: (BOAT) moved unexpectedly: 0.000 (ABOVE WATER) [ping:78 tps:19.93] VL:30
[25-08-2017 12:12:23.372] RedSkilZZ failed Speed: moved too quickly: 0.390 blocks/tick (0.200/-0.528)-n -n [ping:78 tps:19.93] VL:6
[25-08-2017 12:12:32.104] RedSkilZZ failed Fly: (BOAT) moved unexpectedly: -0.477 (-0.078) [ping:73 tps:20.0] VL:8
[25-08-2017 12:12:32.104] RedSkilZZ failed Fly: (BOAT) moved unexpectedly: -0.477 (-0.546) [ping:73 tps:20.0] VL:16
[25-08-2017 12:12:32.254] RedSkilZZ failed Fly: (BOAT) moved unexpectedly: 0.000 (ABOVE WATER) [ping:73 tps:20.0] VL:22
[25-08-2017 12:12:32.254] RedSkilZZ failed Fly: (BOAT) moved unexpectedly: 0.000 (ABOVE WATER) [ping:73 tps:20.0] VL:28
```

**Comment by CeIebrimbor** [comment]
I have another client using 1.12.1 (Paper specifically) reporting the same. Simply using a boat gets you ejected.

**Comment by CeIebrimbor** [comment]
@Cryptite 
This was patched in hotfix 3.3.1-b1, please download and report back if you have issues.
